docs/BROWSERSETUP.md

```diff
@@ -0,0 +1,39 @@
+# Browser Setup for Testing
+It is possible to activate browser specific settings before test execution.  
+E.g. when a special proxy setting is needed for Firefox on a System Under Test (SUT) environment under Linux which is different to the proxy setting under Windows, it is possible to create a JSON-File called **"browserSetup.json** in the directory **src/test/resources** of the actual test project. These specified settings will be used for test execution.
+
+## Example structure of the JSON-File   
+![BrowserSetup](/images/browserSetup.png)
+
+### Browser Setup Elements
+The **browserSetup.json** file consists of several different Browser Setup Elements marked blue in the screenshot.  
+Each Element consists of    
+* an arbitrary chosen element name (e.g. in screenshot "firefox-section-without-os") 
+* an optional Browser Setting entry for the platform (os - marked yellow in the screenshot). 
+* an optional Browser Setting entry for the browser to test (browser - marked yellow in the screenshot). 
+* an optional Browser Setting entry for options (options - marked green in the screenshot).      
+* an optional Browser Setting entry for capabilities (capabilities - marked red in the screenshot). 
+
+
+### Browser Settings
+#### Key "os"  
+When a test should run with OS specific settings the key for "os" must be set, for the specific values please refer to the os specific values on the Selenium web site under [DesiredCapabilities].(https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)  
+When a test should run without OS specific settings means, the specified "Browser Settings" under this "Browser Setup Element" is valid for all OS (currently Windows, Linux, MAC). It is also possible to mix both options for os-specific and not os-specific. An Error will occur when in both options duplicate entries for the same key exists during a test on a given operating system. In the screenshot we have an example for setting the SSL proxy URL and the SSL proxy port for all OS (see screenshot BrowserSetup-Element "firefox-section-without-os")  
```

Comment:
* _OS-specific_
* ~means~
* _macOS_ instead of _MAC_
    * actually, I guess it is "MAC" because that is what Selenium uses – right? In that case, shouldn't the other two supported values also be in all-caps (_WINDOWS_, _LINUX_)?
    * then again, in the screenshot(s), the key _VISTA_ is used, so this might be another source of confusion.
* _An error will occur during a test on a given operating system, if duplicate entries for the same key exist._
* In the screenshot, the "BrowserSetup-Element" is actually called "firefox-section-Without-os" (note capital 'W').